graphql/resolvers.go

```diff
@@ -1064,6 +1067,49 @@ func (r *mutationResolver) SaveSubscription(ctx context.Context, subscription re
 	return true, nil
 }
 
+func (r *mutationResolver) UpdateUserSettings(ctx context.Context, userSettings *restModel.APIUserSettings) (bool, error) {
+	usr := route.MustHaveUser(ctx)
+	adminSettings, err := evergreen.GetConfig()
+	if err != nil {
+		return false, errors.Wrap(err, "Error retrieving Evergreen settings")
+	}
+	changedSettings, err := restModel.ApplyUserChanges(usr.Settings, *userSettings)
+	if err != nil {
+		return false, InternalServerError.Send(ctx, fmt.Sprintf("problem applying user settings: %s", err))
```

Comment:
i think the `err` argument must be `err.Error()` so that the error message is included in error

Reply:
the format code is smart enough to extract the error text, so this works correctly